internal/pkg/agent/cmd/install.go

```diff
@@ -28,6 +29,10 @@ const (
 	flagInstallDevelopment            = "develop"
 	flagInstallNamespace              = "namespace"
 	flagInstallRunUninstallFromBinary = "run-uninstall-from-binary"
+
+	flagInstallADUser  = "ad-user"
+	flagInstallADGroup = "ad-group"
+	flagInstallADPass  = "ad-password"
```

Comment:
Why make this a specific Windows feature? Wouldn't it be better to just allow `user` and `group` for all platforms? `password` is only needed for Windows.

Reply:
i agree, let's see if i can squeeze this in as this is planned for 8.17. if not i can do it iteratively

Reply:
added support for mac and linux, tested on both including switch from privileged to unprivileged. 
tests TBD still